src/dotnet/Chat.UI.Blazor/Components/RightClickChatMenu/RightClickMenuRecord.razor

```diff
@@ -0,0 +1,55 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@using BlazorContextMenu
+@inherits ComputedStateComponent<Symbol>
+
+@{
+    var isRecording = RecordingChatId == Chat.Id;
+    var cls = "context-menu-item important-item " + (isRecording ? "on" : "off");
```

Comment:
`context-menu-item important-item` part looks constant. How about moving it into markup?

Reply:
important-item class highlights red items like delete